readme.md

```diff
@@ -26,8 +28,8 @@ For more information, please refer to the [documentation].
 ## 📦 Features
 
 * **Cross-platform**: runs on Windows, macOS, and Linux!
-* **Cloud native**: supports [Docker], [Azure], [AWS], [Google Cloud], [Alibaba Cloud]
-* **Offline support**: [Mirror a NuGet server] to speed up builds and enable offline downloads
+* **Cloud native**: supports [Docker][Docker doc link], [Azure][Azure doc link], [AWS][AWS doc link], [Google Cloud][GCP doc link], [Alibaba Cloud][Aliyun doc link]
```

Comment:
Please remove Azure for now as that broke with the .NET upgrade. We're working on implementing the new APIs but it's not done yet.